pkg/types/relayer.go

```diff
@@ -91,6 +117,8 @@ type ChainService interface {
 	// Transact submits a transaction to transfer tokens.
 	// If balanceCheck is true, the balance will be checked before submitting.
 	Transact(ctx context.Context, from, to string, amount *big.Int, balanceCheck bool) error
+	// Replay is an emergency recovery tool to re-process blocks starting at the provided fromBlock
+	Replay(ctx context.Context, fromBlock uint64, args map[string]any) error
```

Comment:
Can we use a string here so that we have the flexibility to pass a hash or whatever else for each chain family?
```suggestion
	Replay(ctx context.Context, fromBlock string, args map[string]any) error
```